torchgeo/datasets/geo.py

```diff
@@ -342,7 +346,11 @@ class RasterDataset(GeoDataset):
     #: ``start`` and ``stop`` groups.
     date_format = "%Y%m%d"
 
-    #: True if dataset contains imagery, False if dataset contains mask
+    #: True if the dataset contains source data, such as imagery. False if the dataset
+    #: contains target data, such as a mask. This is the same as Kornia. When multiple
+    #: datasets with different keys are combined and the same key is used for multiple
+    #: datasets, for example 2 "image" and 1 "mask", the channels will be stacked so
+    #: that there's still a single value for that key.
```

Comment:
Does this work:
> The default behavior when multiple datasets with different keys are combined and the same  key is used for multiple datasets, for example 2 "image" and 1 "mask", is that the channels will be stacked so that there's still a single value for that key. However, this behavior can be changed by specifying adifferent ``collate_fn``.

? Or is the documentation now too long for you?

Reply:
I committed the change for your review.